proposals/Template-Instantiation.md

```diff
@@ -44,27 +44,30 @@ partial interface HTMLTemplateElement {
 
 Concretely, use case (1) is addressed by the component instancing a template as follows:
 
-```
+``` js
  // shadowRoot is the shadow root of a contact card component
  shadowRoot.appendChild(template.createInstance());
 ```
 
 Use case (2) is addressed as follows:
 
-```
-// Template content is `'<section><h1>{{name}}</h1>Email: <a href="mailto:{{email}}">{{email}}</a></section>'`
+``` js
+// Template content is `<section><h1>{{name}}</h1>Email: <a href="mailto:{{email}}">{{email}}</a></section>`
 shadowRoot.appendChild(template.createInstance({name: "Ryosuke Niwa", email: "rniwa@webkit.org"}));
 ```
 
 When `createInstance` is called with a JavaScript object, we automatically substitute every mustache syntax with the corresponding value of the property in the object. The resultant DOM would look as though we parsed the following HTML:
 
-```
-<section><h1>Ryosuke Niwa</h1>Email: <a href="mailto:rniwa@webkit.org”>rniwa@webkit.org</a></section>
+``` html
+<section>
+    <h1>Ryosuke Niwa</h1>
```

Comment:
Please don't reformat examples like this. Whitespaces after h1 Before "Email:" would not be collapsed so it would affect the final DOM tree.